packages/js-example-app/src/index.server.ts

```diff
@@ -0,0 +1,233 @@
+                max-length="140"
+                has-label="true"
+                label="Label-text"
+                label-appendix="(Appendix)"
+                has-icon="true"
+                icon="blr360"
+                aria-label="InputFieldText"
+                name="InputFieldText"
+                input-field-text-id="Input Id"
+              >
+              </blr-input-field-text>
+            </div>
+
+            <div class="component">
+              <p>Input Field Number</p>
+              <blr-input-field-number
+                value="1000"
+                size-variant="md"
+                stepper-variant="vertical"
+                placeholder="Placeholder-text"
+                fraction-digits="10"
+                total-digits="10"
+                has-unit="true"
+                unit-position="prefix"
+                unit="kg"
+                step="1"
+                has-label="true"
+                label="Label-text"
+                label-appendix="(Appendix)"
+                input-field-number-id="test-id"
+                theme="Light"
+                name="InputFieldNumber"
+                decimals="10"
+                leading-zeros="3"
+              ></blr-input-field-number>
+            </div>
+
+            <div class="component">
+              <p>Textarea</p>
+              <blr-textarea
+                theme="Light"
+                size-variant="md"
+                resize="both"
+                cols="40"
+                rows="4"
+                placeholder="Placeholder-text"
+                value=""
+                min-length="10"
+                max-length="140"
+                has-label="true"
+                label="Label-text"
+                label-appendix="(Appendix)"
+                has-counter="true"
+                warning-limit-type="warningLimitInt"
+                warning-limit-int="105"
+                aria-label="Text Area"
+                text-area-id="#textAreaId"
+                name="Text Area"
+              >
+              </blr-textarea>
+            </div>
+
+            <div class="component">
+              <p>Tooltip</p>
+
+              <blr-tooltip
+                theme="Light"
+                placement="top"
+                elevation="true"
+                has-arrow="true"
+                offset="20"
+                message="Message-text"
+              >
+                <blr-button-text theme="Light" variant="secondary" size="md" label="Hover me"></blr-button-text>
+              </blr-tooltip>
+            </div>
+
+            <div class="component">
+              <p>Radio Input</p>
+              <blr-radio
+                theme="Light"
+                size-variant="md"
+                value=""
+                label="Label"
+                aria-label=""
+                radio-id="radioId"
+                name="Radio Button"
+              >
+              </blr-radio>
+            </div>
+
+            <div class="component">
```

Comment:
Currently not working. Need to explore the reason.

Reply:
Noted in new [Issue](https://github.com/deven-org/boiler/issues/1122#issue-2348276750). See point **B)**